packages/babel-plugin/src/__tests__/module-traversal.test.ts

```diff
@@ -328,6 +328,46 @@ describe('module traversal', () => {
     expect(actual).toIncludeMultiple(['{color:red}', '{background-color:blue}']);
   });
 
+  it('handles template literal imported from external modules', () => {
```

Comment:
nit 
`'#id-selector'` is a StringLiteral, maybe we should change it to
"handles template literal with imported selectors from external modules"